File: article-api/src/main/kotlin/com/sns/article/reaction/domain/ReactionTarget.java
package com.sns.article.reaction.domain;

public class ReactionTarget {
	private final ReactionTargetType type;
	private final Long id;

	public ReactionTarget(ReactionTargetType type, Long id) {
		this.type = type;
		this.id = id;
	}

	public ReactionTargetType getType() {
		return type;
	}

	public Long getId() {
		return id;
	}
}

File: article-api/src/main/kotlin/com/sns/article/component/reaction/domains/ReactionTarget.kt
package com.sns.article.component.reaction.domains

import org.springframework.data.relational.core.mapping.Column

data class ReactionTarget(
    @field:Column(value = "target_type") val type: ReactionTargetType,
    @field:Column(value = "target_id") val id: Long
)
